libs/ballot-interpreter-nh-next/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@votingworks/ballot-interpreter-nh-next",
+  "version": "1.0.0",
+  "description": "",
+  "main": "./build/index.js",
+  "types": "./build/index.d.ts",
+  "scripts": {
+    "type-check": "tsc --build",
+    "build": "pnpm build:addon && pnpm build:ts",
+    "build:addon": "cargo-cp-artifact -nc build/addon.node -- cargo build --message-format=json-render-diagnostics --release",
+    "build:ts": "tsc --build tsconfig.build.json",
+    "clean": "cargo clean && rm -rf build tsconfig.tsbuildinfo tsconfig.build.tsbuildinfo",
+    "lint": "pnpm type-check && eslint .",
+    "lint:fix": "pnpm type-check && eslint . --fix",
+    "install": "pnpm build:addon",
```

Comment:
This is something that `neon` set up, and I think I agree. Without this you'd have to explicitly build this package for any package that uses it. We don't have to do that with the others because they use TypeScript project references to automatically build dependencies.

Reply:
Ah ok that makes sense!